Flow.Launcher/MainWindow.xaml

```diff
@@ -68,8 +69,8 @@
                          Margin="18,0,56,0">
                         <TextBox.Text>
                             <MultiBinding Converter="{StaticResource QuerySuggestionBoxConverter}">
-                                <Binding ElementName="QueryTextBox" Path="Text"/>
-                                <Binding ElementName="ResultListBox" Path="SelectedItem"/>
+                                <Binding ElementName="QueryTextBox" Path="Text" Mode="OneWay"/>
+                                <Binding ElementName="ResultListBox" Path="SelectedItem" Mode="OneWay"/>
```

Comment:
how come we are adding binding to one way for this change?

Reply:
Sorry, that was added incidently. The svg seems make by designer break every time after debugging, so I just try to check whether this led to that. I will remove it.

Reply:
Found that why I changed here to OneWay. This is the auto suggest factory, so we won't need to let user to change something here to move change to the query text and result list box.

Reply:
ok no problems thanks.